### PR TITLE
added dataclass SensorInfo

### PR DESCRIPTION
**Pull request by VictoriaSer**

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -7,6 +8,50 @@
...
+            raise TypeError("'model' should be a string")
+        
+        if self.model == "":
+            raise ValueError("'model' should not be empty")
+        
+
+        if not isinstance(self.firmware_version, int):
+            raise TypeError("'firmware_version' should be integer")
+        
+        if self.firmware_version == "":
```
Comment (ddoroshevskyi) [inline conftest.py]: Testing for a non-empty string is suitable for string properties but here we've got an integer. We'd like to test this for validity from sensor's standpoint. That would be checking that the firmware version is between 10 and 15, both ends included. You could test this with the following code:
```
10 <= self.firmware_version <= 15
```
Of course, you'll need to update the error message in the `ValueError` to reflect this new check.

File: conftest.py [hunk 2]
```diff
@@ -7,6 +8,50 @@
...
+            raise ValueError("'firmware_version' should not be empty")
+        
+
+        if not isinstance(self.reading_interval, int):
+            raise TypeError("'reading_interval' should be float")
```
Comment (ddoroshevskyi) [inline conftest.py]: This error message should have `int` instead of `float`.

File: conftest.py [hunk 3]
```diff
@@ -7,6 +8,50 @@
...
+        
+        if self.reading_interval == "":
```
Comment (ddoroshevskyi) [inline conftest.py]: Same thing as in my first comment applies here. We're interested in checking that this is a valid value from sensor's standpoint. That would be seeing that it's equal or more than 1.

File: sensor_test.py [hunk 4]
```diff
@@ -1,25 +1,26 @@
...
     assert isinstance(sensor_model, str), "Sensor model is not a string"
 
-    sensor_firmware_version = sensor_info.get("firmware_version")
+    sensor_firmware_version = sensor_info.firmware_version
     assert isinstance(sensor_firmware_version, int), "Sensor firmware_version is not an int"
 
-    sensor_reading_interval = sensor_info.get("reading_interval")
-    assert isinstance(sensor_reading_interval, int), "Sensor reading_interval is not an int"
+    sensor_reading_interval = sensor_info.reading_interval
+    assert isinstance(sensor_reading_interval, int), "Sensor reading_interval is not a float"
```
Comment (ddoroshevskyi) [inline sensor_test.py]: Sensor reading interval is an int, so there's no need to edit this message.

**Review by ddoroshevskyi — CHANGES_REQUESTED** [review]

File: sensor_test.py (hunk 4 above)
Comment (VictoriaSer) [reply sensor_test.py:23]: image

there is an error meggase, that's why I removed get

File: conftest.py (hunk 1 above)
Comment (VictoriaSer) [reply conftest.py]: thank you very much! 
done

File: conftest.py (hunk 2 above)
Comment (VictoriaSer) [reply conftest.py]: sorry, my fault!
fixed!

File: conftest.py (hunk 3 above)
Comment (VictoriaSer) [reply conftest.py]: this one also done!

File: sensor_test.py (hunk 4 above)
Comment (VictoriaSer) [reply sensor_test.py]: my mistake!
fixed the error message!

File: conftest.py [hunk 5]
```diff
@@ -7,6 +8,50 @@
 
 log = logging.getLogger(__name__)
 
+@dataclass
+class SensorInfo:
+    name: str
+    hid: str
+    model: str
+    firmware_version: int
+    reading_interval: float
```
Comment (ddoroshevskyi) [inline conftest.py]: This type hint should read `int` instead of `float`. Just noticed it, sorry I didn't catch it yesterday.

**Review by ddoroshevskyi — CHANGES_REQUESTED** [review]

File: conftest.py (hunk 5 above)
Comment (VictoriaSer) [reply conftest.py]: Thank You!
I should have also chacked it((

**Pull request merged**